src/main/js/helpers/axis.js

```diff
@@ -683,16 +677,9 @@ const getYAxisWidth = (id, config) => {
  * @param {object} config - config object derived from input JSON.
  * @returns {number} label width
  */
-const getY2AxisWidth = (config) => {
-    if (config.padding.hasCustomPadding) {
-        return config.padding.right;
-    }
-    return (
-        (hasY2Axis(config.axis)
-            ? getYAxisWidth(constants.Y2_AXIS, config)
-            : 20) + config.padding.right
-    );
-};
+const getY2AxisWidth = (config) =>
```

Comment:
Changes look great in Graph construct but for Gantt construct its still a bit broken. I bet that's because we are using `hasCustomPadding` property there as well. 

With adding the property as 
```
top: 10,
bottom: 5,
left: 30,
right: 50
```
**Gantt**
Gantt with `padding` property used:
![image](https://user-images.githubusercontent.com/4908886/65251799-94b3db80-dabd-11e9-8152-055b945e536a.png)

Gantt without `padding` property used:
![image](https://user-images.githubusercontent.com/4908886/65251895-b4e39a80-dabd-11e9-90d8-114e2e77c564.png)